utils.go

```diff
@@ -13,19 +13,15 @@ import (
 	cid "github.com/ipfs/go-cid"
 )
 
-func ScanForLinks(br io.Reader) ([]cid.Cid, error) {
-	var out []cid.Cid
-	if err := scanForLinksRec(br, func(c cid.Cid) {
-		out = append(out, c)
-	}); err != nil {
-		return nil, err
-	}
+const maxCidLength = 100
```

Comment:
Will this apply to embedded identity multihash links?

Reply:
@ianopolous yes, though only if youre using this method. Is that an issue? I guess we can make it a var instead of a const so you can tweak it in your own applications

Reply:
We only interact over the http api. So hopefully that doesn't affect us then? I can't think of a use case where it actually matters unless pinning could end up calling this to get links and then error?

Reply:
It looks like this library is used in go-ipfs indirectly through go-ipld-cbor: https://github.com/ipfs/go-ipld-cbor/blob/master/go.mod#L12